### ISSUE TITLE
Where is the ignoreNetworkInterfaces branch code?

### ISSUE THREAD

**Issue by janyyy** [fmcejudo/quarkus-eureka#60]
@fmcejudo There is a requirement before, when registering by hostname, always get the Docker virtual NIC, later you wrote a configuration in ignoreNetworkInterfaces to ignore the NIC, where did this branch code merge to? Where can I find it there? Is it in the latest master branch?

**Comment by fmcejudo** [comment]
Hi @janyyy , I didn't mean to remove any feature. I have just moved to the `jakarta` namespace and update some dependencies. Also adding the heartbeat which we used to implement ourself.

At some point we were working in this: https://github.com/fmcejudo/quarkus-eureka/issues/37

Whose implementation was in here:  https://github.com/fmcejudo/quarkus-eureka/pull/38

But this never went into `master` branch. Do you think you might need it?

If so, would you mind to give more details of the use case?

Thanks in advance.

**Comment by janyyy** [comment]
@fmcejudo By default, quarkus-eurake will take the first NIC br-2b644052f6b2 for registration, but my NICs starting with br- and docker0 are generated by the docker container and are not connected to my local machine. I need to register the eth0 NIC with the corresponding hostname, otherwise the services will not communicate with each other. I would like you to add that configuration to the master branch, this is useful and it would be nice to add regular expressions. For support of regular expressions to ignore NICs you can refer to my code, which is necessary because the Dokcer container generates
![image](https://github.com/fmcejudo/quarkus-eureka/assets/27274865/194bcc02-79b1-451b-883b-310aea5b775c)
![image](https://github.com/fmcejudo/quarkus-eureka/assets/27274865/092ce467-2fc1-47b7-b3f1-49cb5e51dbba)
![image](https://github.com/fmcejudo/quarkus-eureka/assets/27274865/d652f044-36a0-430a-8cb8-d83b3dbfdb17)

**Comment by janyyy** [comment]
@fmcejudo  And we are now using JDK11 and Quarkus2.xxx, I see that you are upgrading Quarkus3.0 Can you keep a separate branch for Quarkus2.xxx

**Comment by fmcejudo** [comment]
@janyyy  thanks for the details and yes, you are completely right, I have to keep both versions. So my concern is about to maintain version 0.0.X with Quarkus 2 and 1.X.X to Quarkus 3.

I will work on that this week and sorry. Up to now the best option for you then is to use `0.0.16` yet.

**Comment by janyyy** [comment]
@fmcejudo thanks, I previously downloaded the ignore_network_interfaces branch code locally, I upgraded quarkus version 2.12.3.Final and made the ignoreNetworkInterfaces logic support regular expressions, but I recompiled and reported errors in the release. Why is this?
![image](https://github.com/fmcejudo/quarkus-eureka/assets/27274865/68537447-6815-4c0a-9e96-3e65f8edacaa)

**Comment by janyyy** [comment]
@fmcejudo this is my dependency
![image](https://github.com/fmcejudo/quarkus-eureka/assets/27274865/294ed226-f0b2-417e-ad7a-7a88c98b6b4d)

**Comment by janyyy** [comment]
> @janyyy thanks for the details and yes, you are completely right, I have to keep both versions. So my concern is about to maintain version 0.0.X with Quarkus 2 and 1.X.X to Quarkus 3.
> 
> I will work on that this week and sorry. Up to now the best option for you then is to use `0.0.16` yet.

@fmcejudo thanks，Version 0.0.16 has ignore-network-interfaces configuration ？

**Comment by fmcejudo** [comment]
I have just released a new version for `Quarkus 2` with that configuration we had. 

The branch is `main-quarkus-2` and it should be released in a version `0.0.19`

```
<dependency>
    <groupId>com.github.fmcejudo</groupId>
    <artifactId>quarkus-eureka-deployment</artifactId>
    <version>0.0.19</version>
</dependency>
```

it should be available sooner than later. Though you might pull down the `main-quarkus-2` branch and `mvn clean install`  to build it locally.

Let see if it works. Thanks and sorry for that.

**Comment by janyyy** [comment]
@fmcejudo  thank you
